source/code/plugins/in_sudo_tail.rb

```diff
@@ -101,6 +101,10 @@ def receive_log(line)
     end
  
     def run_periodic
+      @paths = @path.split(' ').each {|path|
+        $log.info "Following tail of #{path}"
```

Comment:
If you don't want redundant repetitive logs, consider using something like warn_once in omslog.rb but for info level.